README.md

```diff
@@ -135,23 +135,33 @@ There is also a [tapioca-wrapper](https://github.com/vintasoftware/tapioca-wrapp
 If you have [Docker](https://docs.docker.com/engine/installation/) (with [Docker Compose](https://docs.docker.com/compose/install/)) and make, just run:
 
 ```console
-$ docker-compose up -d --build
-$ docker-compose run --rm jarbas python manage.py migrate
-$ docker-compose run --rm jarbas python manage.py ceapdatasets
+make run.devel
+```
+
+or
+
+```console
+docker-compose up -d --build
+docker-compose run --rm jarbas python manage.py migrate
+docker-compose run --rm jarbas python manage.py ceapdatasets
+docker-compose run --rm jarbas python manage.py collectstatic --no-input
 ```
 
 You can access it at [`localhost:80`](http://localhost:80/). However your database starts empty and you still have to collect your static files:
```

Comment:
> As we are extracting variables for development and production environment, is it easy to make make run.devel to be available in localhost:8000 (Django's default)?

It is easy! We just need to change docker-compose.yml (development file)

> The sentence makes no sense anymore, since collectstatic happened in the previous step.
> Maybe rephrase it to explain that either one can load the sample data from the repo just to get started, or run Rosie to get the full dataset (maybe moving a sentence that is a few lines below to this point).

I agree. I will change it! :)